api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -120,7 +121,7 @@ public Workspace getByName(String ns, String name) {
   @Override
   public Workspace getRequired(String ns, String firecloudName) {
     Workspace workspace = get(ns, firecloudName);
-    if (workspace == null) {
+    if (workspace == null || (workspace.getWorkspaceActiveStatusEnum() == WorkspaceActiveStatus.DELETED)) {
```

Comment:
I would probably flip this to != ACTIVE, it is unlikely we'd want other statuses showing up here, if we added any others in the future